pkg/tfpfbridge/schemashim/attr_schema.go

```diff
@@ -92,8 +92,10 @@ func (s *attrSchema) Elem() interface{} {
 		lT := t.(tftypes.List)
 		var schema shim.Schema = &typeSchema{lT.ElementType}
 		return schema
+	case t.Is(tftypes.Object{}):
```

Comment:
This needed to be filled in for SingleNestedAttributes to work.

Reply:
Add this as documentation to the code?